Assets/Scripts/Framework/SceneSwitcher.cs

```diff
@@ -8,6 +10,8 @@ public sealed class SceneSwitcher : MonoBehaviour
         [SerializeField] private bool loadSceneInAwake;
         [SerializeField] private string sceneToLoad;
 
+        public float Progress;
```

Comment:
```suggestion
        priavte float _progress;
```
Deze float wordt alleen hier gebruikt, hoeft niet public te zijn.